front/components/sparkle/navigation.tsx

```diff
@@ -139,6 +144,17 @@ export const subNavigationAdmin = ({
       subMenuLabel: current === "developers" ? subMenuLabel : undefined,
       subMenu: current === "developers" ? subMenu : undefined,
     });
+    if (isDevelopmentOrDustWorkspace(owner)) {
+      nav.push({
+        id: "assistants",
+        label: "Assistants Manager",
```

Comment:
NIT: Assistant Builder?

Reply:
took it from the design but agreed builder is more consistent